components/rhs_root_post/index.js

```diff
@@ -18,6 +21,7 @@ function mapStateToProps(state, ownProps) {
     return {
         enableEmojiPicker,
         enablePostUsernameOverride,
+        isEmbedVisible: isEmbedVisible(state, ownProps.post.id),
```

Comment:
Maybe something we need to think about on how to test this container so that we can validate whether the component is getting the right props from the container.